pkgs/development/libraries/eccodes/default.nix

```diff
@@ -1,39 +1,52 @@
-{ fetchurl, lib, stdenv
-, cmake, netcdf, openjpeg, libpng, gfortran
-, enablePython ? false, pythonPackages
+{ fetchurl
+, lib
+, stdenv
+, cmake
+, netcdf
+, openjpeg
+, libpng
+, gfortran
+, perl
+, enablePython ? false
+, pythonPackages
 , enablePosixThreads ? false
-, enableOpenMPThreads ? false}:
+, enableOpenMPThreads ? false
+}:
+
 with lib;
 stdenv.mkDerivation rec {
```

Comment:
```suggestion
stdenv.mkDerivation rec {
```

Reply:
It's actually used in `meta` and for `optionals` in `propagatedBuildInputs`, I made the `optionals` explicit and added `with lib;` to meta as seems to be common practice in other packages.